sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorTraceExporter.java

```diff
@@ -681,11 +680,11 @@ private void setExtraAttributes(TelemetryItem telemetry, Map<String, String> pro
                 return;
             }
             // special case mappings
-            if (key.equals(SemanticAttributes.ENDUSER_ID) && value instanceof String) {
+            if (key.equals("enduser.id")) {
                 telemetry.getTags().put(ContextTagKeys.AI_USER_ID.toString(), (String) value);
                 return;
             }
-            if (key.equals(SemanticAttributes.HTTP_USER_AGENT) && value instanceof String) {
+            if (key.equals("http.user_agent")) {
                 telemetry.getTags().put("ai.user.userAgent", (String) value);
```

Comment:
is it intentional to remove the instanceof check?

Reply:
Added it back.